SECURITY.md

```diff
@@ -0,0 +1,165 @@
+
+- an extension that is not commonly used
+- a particular type of configuration that is used only in narrow
+  specific circumstances
+- relies on old version of a third-party library being used
+- code, or patterns of code, that are known to be used infrequently
+- code that is very old, or extremely uncommon (and so is used
+  infrequently)
+
+This kind of issues usually will have a CVE number, unless the required
+configuration is particularly exotic to the point it's not practically
+usable.
+
+### Low severity
+
+This issue allows theoretical compromise of security, but practical
+attack is usually impossible or extremely hard due to common practices
+or limitations that are virtually always present or imposed.
+
+This also includes problems with configuration, documentation, and other
+non-code parts of the PHP project that may mislead users, or cause them
+to make their system, or their code less secure.
+
+Issues that can trigger unauthorized actions that do not seem to be
+useful for any practical attack can also be categorized as low severity.
+
+Security issues, that are present only in unstable branches, belong to
+this category, too. Any branch that has no stable release, is per se not
+intended for the production use.
+
+Low severity issues usually do not need to have CVE and may, at the
+discretion of the PHP developers, be disclosed publicly before the fix
+is released or available.
+
+### Not a security issue
+
+We do not classify as a security issue any issue that:
+
+- requires invocation of specific code, which may be valid but is
+  obviously malicious
+- requires invocation of functions with specific arguments, which may
+  be valid but are obviously malicious
+- requires specific actions to be performed on the server, which are
+  not commonly performed, or are not commonly permissible for the user
+  (uid) executing PHP
+- requires privileges superior to that of the user (uid) executing PHP
+- requires the use of debugging facilities - ex. xdebug, var_dump
+- requires the use of settings not recommended for production - ex.
+  error reporting to output
+- requires the use of non-standard environment variables - ex.
+  USE_ZEND_ALLOC
+- requires the use of non-standard builds - ex. obscure embedded
+  platform, not commonly used compiler
+- requires the use of code or settings known to be insecure
+- requires the use of FFI
+- requires an open_basedir bypass
+
+## Handling issues
+
+High and medium severity fixes are merged into a security repository and
+merged before the release is tagged.
+
+Low severity fixes are merged immediately after the fix is available and
+handled like all regular bugs are handled consequently. However, release
+managers may choose to pull those fixes into the RC branch after the
+branch is created, and also backport them into security-only release
+branch.
+
+## FAQ
+
+Q. How do I report a security issue?\
+A. Please report it on <http://bugs.php.net>, choosing type "Security".
```

Comment:
```suggestion
A. Please report it on <https://bugs.php.net>, choosing type "Security".
```

Reply:
Good catch! I copied the text directly from the Dokuwiki editor source.